strapi/strapi
# [Blocks] Fix converting list items

### What does it do?

Fix the conversion of list items.

Before, when selecting a different type of block while the selection was in a list, it created the block inside of the list. This created UI and validation errors, because only list items and lists are allowed as children of a list.

Now, the parent list will be split in 2, and the modified block will sit in between the 2 list blocks.

### How to test it?

Convert a list item to another block type, save and publish, you should not get a validation error
